pkg/kubelet/cm/cpumanager/cpu_assignment.go

```diff
@@ -196,6 +196,58 @@ func (s *socketsFirst) sortAvailableCores() []int {
 	return result
 }
 
+type sortCPUsPackedOrSpreadFuncs interface {
+	sortAvailableCores() []int
+}
```

Comment:
Are we sorting CPUs or Cores? We should be consistent with the naming. Can we change this interface to:
```
type availableCPUSorter interface {
	sortAvailableCPUs() []int
}
```

Reply:
Indeed. I think the naming here is confusing. It's sorting CPUs. I make the change.